### DISCUSSION TITLE
make the arms visible when placing Armor_Stand while crouching

### DISCUSSION THREAD

**Discussion by Huge-mistake** [PurpurMC/Purpur#1595] [Feature Requests]
Currently, Purpur provides an option called "[place-with-arms-visible](https://purpurmc.org/docs/purpur/configuration/#place-with-arms-visible)" for Armor_Stand to modify whether the arms are visible.

I currently encounter a problem: some of my players need Armor_Stand without visible arms, as they require this in certain AFK mob farms, while others need Armor_Stand with visible arms. It's a standoff. Is it possible to differentiate whether to display the arms when placing Armor_Stand based on whether it's in sneaking mode or not?

**Comment by Onako2** [comment]
Sounds interesting!

**Comment by RhythmicSys** [comment]
This is a good suggestion for [Purpur Extras](https://github.com/PurpurMC/PurpurExtras) as it would be a lot simpler to do these checks in a plugin environment
